code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -271,7 +280,7 @@ var/list/robot_verbs_default = list(
 /mob/living/silicon/robot/proc/pick_module()
 	if(module)
 		return
-	var/list/modules = list("Standard", "Engineering", "Medical", "Miner", "Janitor", "Service")
+	var/list/modules = list("Standard", "Engineering", "Medical", "Miner", "Janitor", "Service")  //add     >, "Research"<     when it's ready

```

Comment:
So was the research module added?

Reply:
Are you fucking with me right now

Reply:
Hey, I didn't touch this code. It's not on me

Reply:
So did not one single one of my more than one multiple of my commit messages go through?

Reply:
No, I'm sure they all went through. It's just that no one pays attention to them or code reviews